includes/class-cli.php

```diff
@@ -199,4 +199,36 @@ public function reschedule( $args ) {
 
 		WP_CLI::success( 'Rescheduled activity.' );
 	}
+
+	/**
+	 * Move the blog to a new URL.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <from>
+	 *     The current URL of the blog.
+	 *
+	 * <to>
+	 *     The new URL of the blog.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *    $ wp activitypub move https://example.com/ https://newsite.com/
+	 *
+	 * @synopsis <from> <to>
+	 *
+	 * @param array $args The arguments.
+	 */
+	public function move( $args ) {
+		$from = $args[0];
+		$to   = $args[1];
+
+		$outbox_item_id = Move::from_to( $from, $to );
```

Comment:
```suggestion
		$outbox_item_id = Move::account( $from, $to );
```
Should this be `account()`?